packages/dd-trace/src/exporters/common/request.js

```diff
@@ -42,10 +42,19 @@ function urlToOptions (url) {
   return options
 }
 
-function fromUrlString (url) {
-  return typeof urlToHttpOptions === 'function'
-    ? urlToOptions(new URL(url))
-    : urlParse(url)
+function fromUrlString (urlString) {
+  const url = typeof urlToHttpOptions === 'function'
+    ? urlToOptions(new URL(urlString))
+    : urlParse(urlString)
+
+  // Add the 'hostname' back if we're using named pipes
+  if (url.protocol === 'unix:' && url.host === '.') {
+    const udsPath = urlString.replace(/^unix:/, '')
```

Comment:
The idea is that the URL has been stripped down from `unix://./pipe/datadogtrace` to `/pipe/datadogtrace` and we're just setting them to `//./pipe/datadogtrace` instead